src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -872,6 +872,22 @@ begin
     exact hp.2 hx, },
 end
 
+variable {p}
+
+protected lemma is_path.map (hp : (p.map f).is_path) : p.is_path :=
```

Comment:
Can we have the `iff` lemma combining this and the previous in the case that `f` is injective?

Reply:
Added